src/main/java/io/cdap/e2e/utils/CdfHelper.java

```diff
@@ -29,15 +43,96 @@
 public interface CdfHelper {
 
   default void openCdf() throws IOException, InterruptedException {
-    SeleniumDriver seleniumDriver = new SeleniumDriver();
-    RemoteWebDriver driver = createDriverFromSession(seleniumDriver.session(), seleniumDriver.url);
+    RemoteWebDriver driver = createDriverFromSession(SeleniumDriver.session(), SeleniumDriver.url);
     SeleniumDriver.openPage(SeleniumHelper.readParameters("CDFURL"));
     try {
       SeleniumDriver.getDriver().switchTo().alert().accept();
       SeleniumDriver.waitForPageToLoad();
     } catch (NoAlertPresentException Ex) {
       SeleniumDriver.waitForPageToLoad();
     }
+  }
+
+  static int getCountOfNoOfRecordsTransferredToBigQueryIn(String arg0) throws IOException, InterruptedException {
+    int countRecords;
+    countRecords = GcpClient.countBqQuery(SeleniumHelper.readParameters(arg0));
+    BeforeActions.scenario.write("**********No of Records Transferred in table" + SeleniumHelper.readParameters(arg0)
+                                   + "*:"
+                                   + countRecords);
+    Assert.assertTrue(countRecords > 0);
+    return countRecords;
+  }
+
+  static void selectSourceGCS() throws InterruptedException {
```

Comment:
Some methods static, but others are not. Is there a reason why you've implemented it like that?

Reply:
changed to default